mix.exs

```diff
@@ -36,9 +36,10 @@ defmodule ExqUI.MixProject do
     [
       {:exq, ">= 0.16.2"},
       {:exq_scheduler, "~> 1.0", optional: true},
-      {:phoenix_live_view, "~> 0.18"},
+      {:phoenix_live_view, "~> 0.20.12"},
```

Comment:
This was the last version that supported Elixir 12
https://github.com/phoenixframework/phoenix_live_view/commit/83f3f84096423129d4d7e3f25b676997884d7e35

all the mix lock updates were for the live view and html helpers. I tried keeping it to a minimum to make them work